### PR TITLE
[MM-49932] Remove experimental warning about plugins

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
Plugins are no longer in beta or marked as experimental. The codebase should reflect that.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-49932

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Review by noxer — APPROVED** [review]
Yeah, fair enough. Thanks :)

**Review by mickmister — APPROVED** [review]

**Comment by mickmister** [comment]
@hanzei I think this is good to merge. Note that I think we may still say plugins are Beta in the UI or in docs. I don't think they're "experimental" though.

**Comment by hanzei** [comment]
> Note that I think we may still say plugins are Beta in the UI or in docs. I don't think they're "experimental" though.

I've also checked for that and didn't find any mentioned of a `Beta`. Please let me know if you stumble across any mentions of `Beta`.

File: api4/plugin.go [hunk 1]
```diff
@@ -25,8 +23,6 @@ const (
 )
 
 func (api *API) InitPlugin() {
-	mlog.Debug("EXPERIMENTAL: Initializing plugin api")
-
```
Comment (DHaussermann) [inline api4/plugin.go:29]: @hanzei this is already merged but I'm just curious what your intension was here. Should we have kept the Initializing plugin api" portion?

Comment (hanzei) [reply api4/plugin.go:29]: I don't think so. Neither of the other API handlers has a similar log message.

**Pull request merged**